applications/openshift/api-server/api_server_no_adm_ctrl_plugins_disabled/rule.yml

```diff
@@ -32,21 +35,23 @@ ocil_clause: 'No admission plugins are disabled'
 
 ocil: |-
     To verify that the list of disabled admission plugins is empty, run the following command:
-    <pre>$ oc -n openshift-kube-apiserver get configmap config -o json | jq -r '.data."config.yaml"' | jq '.apiServerArguments."disable-admission-plugins"'</pre>
+    <pre>$ <pre>$ oc -n openshift-kube-apiserver get mconfigmap config -o json | jq -r '{{{ jqfilter }}}'</pre>
```

Comment:
It looks like we have an extra `<pre>$` here?

Reply:
thanks for the review! Let me fix those!